.idea/.gitignore

```diff
@@ -0,0 +1,2 @@
+# Default ignored files
+/workspace.xml
```

Comment:
Please remove `.idea` directory.

Reply:
@iizukak Whaaaaat's author mentioned `inquirer`, and I checked it and it is similar to what `Whaaaaat` can do. Thus, I used it.

Reply:
`.idea` is removed.